browser/src/slideshow/SlideShowPresenter.ts

```diff
@@ -721,7 +727,12 @@ class SlideShowPresenter {
 		if (!this._onPrepareScreen(true))
 			// opens full screen, has to be on user interaction
 			return;
-
+		// disable present in console if on present in window executed
+		this._map.fire('commandstatechanged', {
```

Comment:
this code is duplicated, can we create method: `enablePresenterConsole(enable: boolean)` ?

Reply:
Sure